### PR TITLE
project: Release v0.1.0-alpha.13

### PR DESCRIPTION
**Pull request by sreis**
I've already built and pushed the docker image. After merging this PR we need to create and push the tag.

### CODE REVIEW

File: RELEASE-NOTES.adoc [hunk 1]
```diff
@@ -19,6 +19,13 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v0.1.0-alpha.13
+
+=== Improvements
+
+* Add option to configure default number of load balancers in the EdgeLB pools. The default is set to 1. A custom value can be specified using the `--edgelb-defaul-pool-size` command line flag. (https://github.com/mesosphere/dklb/pull/46[#46])
+* Add option to configure load balancer instance placement for the target EdgeLB Pool. The default is no constraint is specified. A custom value can be speficied using `.constraints` field in `kubernetes.dcos.io/dklb-config` annotation. (https://github.com/mesosphere/dklb/pull/47[#47])
```
Comment (jimmidyson) [inline RELEASE-NOTES.adoc]:
```suggestion
* Add option to configure load balancer instance placement for the target EdgeLB Pool. By default, no constraint is specified. A custom value can be specified using the `constraints` field in the `kubernetes.dcos.io/dklb-config` annotation. (https://github.com/mesosphere/dklb/pull/47[#47])
```

File: RELEASE-NOTES.adoc [hunk 2]
```diff
@@ -19,6 +19,13 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v0.1.0-alpha.13
+
+=== Improvements
+
+* Add option to configure default number of load balancers in the EdgeLB pools. The default is set to 1. A custom value can be specified using the `--edgelb-defaul-pool-size` command line flag. (https://github.com/mesosphere/dklb/pull/46[#46])
```
Comment (jimmidyson) [inline RELEASE-NOTES.adoc]:
```suggestion
* Add option to configure default number of load balancers in the EdgeLB pools. By default, the number of load balancers in the EdgeLB pool is set to 1. A custom value can be specified using the `--edgelb-defaul-pool-size` command line flag. (https://github.com/mesosphere/dklb/pull/46[#46])
```

File: RELEASE-NOTES.adoc (hunk 1 above)
Comment (sreis) [reply RELEASE-NOTES.adoc]: Forced pushed with this change.

File: RELEASE-NOTES.adoc (hunk 2 above)
Comment (sreis) [reply RELEASE-NOTES.adoc]: Forced pushed with this change.

**Pull request merged**